yogstation/code/modules/donor/unique_donator_items.dm

```diff
@@ -74,19 +78,45 @@ GLOBAL_DATUM_INIT(donator_gear, /datum/donator_gear_resources, new)
 			qdel(S)
 			continue
 		donor_items += S
+	for(var/Plushtype in subtypesof(/obj/item/toy/plush)) //generate our plushies
+		var/datum/donator_gear/P = new
+		var/obj/item/toy/plush/myplush = new Plushtype
+		if(myplush.donor_blacklist) //don't give dehydrated carp plushie to donors
+			qdel(P)
+			qdel(myplush)
+			continue
+		if(myplush.name == "coder plush") //DON'T GIVE THE PLAYERS THE CODER PLUSH
```

Comment:
You don’t need this check

Reply:
subtypesof wont include the parent type if you’ve done it correctly. And even if it included the parent type this should be checked using the path and not a magic string

Reply:
I'm going to emphasize that you don't need this check.
subtypesof won't grab the parent type.
I'd be happy to get you a code example if you don't believe me